fvtr/systemtap/systemtap.exp

```diff
@@ -0,0 +1,36 @@
+#! /usr/bin/expect
+# Copyright 2019 IBM Corporation
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+#
+# Check if Glibc has SDTs defined.
+
+source shared.exp
+
+if { $env(AT_MAJOR_VERSION) < 11.0 } {
+    printit "Skipping: systemtap isn't enabled on version older than AT 11.0"
+    exit $ENOSYS
+}
+set lib "lib64"
+if { $TARGET32 } {
+    set lib "lib"
+}
+set readelf_path [exec which readelf]
+set sdt [exec $readelf_path --notes  "$env(AT_DEST)/$lib/libpthread.so" | \
+	     grep sdt | wc -l]
```

Comment:
Don't you need to use `eval` here, e.g. https://github.com/advancetoolchain/advance-toolchain/blob/master/fvtr/shared.exp#L402

Reply:
Also, 'sdt' is a simple string, are you sure this won't match anything else?  Maybe replace it with stapsdt?

Reply:
@gftg85 I replaced `sdt` with `stapsdt`.